### DISCUSSION TITLE
Extend objection to support desktop apps

### DISCUSSION THREAD

**Discussion by 0vulns** [sensepost/objection#663] [Ideas]
Hi!, I was recently exploring frida with desktop apps so i was thinking to add desktop apps (MacOS for now) support for objection.

**Comment by leonjza** [comment]
Cool! There is definitely a world where objection is no longer tied to just mobile devices :)

**Reply by 0vulns** [reply]
Initially, I entertained the idea of building a separate toolkit for desktop applications 🤣. However, I was inspired by the efforts of objection team to make it inherently extensible, and that led me to this current idea.
I already started working on it :) 

![image](https://github.com/sensepost/objection/assets/12600640/d984df68-5acf-4a02-a9a2-d99ea1bdad51)

**Comment by 0vulns** [comment]
**Update:**

1. env 
2. binary info 

are fully supported now for macos 💯 

![image](https://github.com/sensepost/objection/assets/12600640/a89d111c-7162-486c-bbd3-dccb1686f766)

**Comment by 0vulns** [comment]
**Update:**
3. Bypass ssl pinning  💯 
![image](https://github.com/sensepost/objection/assets/12600640/4accd2e6-0803-4184-8ca4-b37e8d1932e1)
![image](https://github.com/sensepost/objection/assets/12600640/82981836-590b-41d3-bda6-55f01248a615)
